### PR TITLE
Topic/aashish/ipv6

### PR DESCRIPTION
**Pull request by initconf**
This is NaokiEto's branch with slight modification made by me to strip out vlan tags. 

I'd love if you can merge this branch with master. 

Code has been running STABLE at LBNL for > 4 years now. We don't use indexes. Performance has been fantastic on memory and CPU.

For exmaple here is current production process stats:

tm         932    5.9  0.9 1390028 440792  -  Is   26Jul17   19864:27.43 /usr/local/bin/timemachine -c /usr/local/etc/timemachine.cfg

and here is top (to give memory and CPU footprint - of this running since Jul 2017)

PID USERNAME       THR PRI NICE   SIZE    RES STATE   C   TIME    WCPU COMMAND
 932 tm               4  20    0  1357M   430M nanslp 16 331.1H   3.27% timemachine

Here is stats 0.02% packet drops since July, 2017:

Apr 20 12:47:04 903929 0 524570585 411375243 0.02% 0 2206808700740 1924700605985703 471.52 244313367515 65570897949722 28.66 0.0 0.0 0 0 0 0 0

Basically processing 1.924 Petabytes of data in the duration and dropping 384GB while shrinking down 1.924Petabytes into 65.57TB on disk - we haven't yet identified a connection we were looking for in the data and not found it to reconstruct back

### CODE REVIEW

**Comment by initconf** [comment]
Basically processing 1.924 Petabytes of data in the duration and dropping 384GB while shrinking down 1.924Petabytes into 65.57TB on disk - we haven't yet identified a connection we were looking for in the data and not found it to reconstruct back

**Comment by timwoj** [comment]
It looks like the changes here were merged as part of https://github.com/zeek/time-machine/commit/33b85b16b58e5d5178a276c4ee618ba74045601e. Is that correct? Can this PR be closed?

I'm just going through all of the github projects in the zeek org and noting anything that we might have missed, and this one caught my eye.

**Comment by ckreibich** [comment]
That's my read too, Tim. Let me close this.

**Comment by initconf** [comment]
[+Case Larsen to the thread]

Yes, thats correct - these were merged a while back and I believe ***@***.***
has now has merge master privileges on TimeMachine so we are actually good here.

Thanks,
Aashish
